src/hooks/useMultiSelect.stories.mdx

```diff
@@ -41,22 +39,134 @@ export default function MultiSelectWithHookExample() {
   const multiSelectId = "multi-select-1";
   // Without this hook, you'd have to add handlers for onChange and onClear,
   // and also manage the selectedItems state yourself.
-  const { onChange, onClear, selectedItems } = useMultiSelect(
-    multiSelectId,
-    items
+  const { onChange, onClear, selectedItems } = useMultiSelect();
+
+  return (
+    <MultiSelect
+      id={multiSelectId}
+      variant="listbox"
+      items={items}
+      selectedItems={selectedItems}
+      onChange={(selectedItem) => onChange(selectedItem.id, multiSelectId)}
+      onClear={() => {
+        onClear(multiSelectId);
+      }}
+    />
   );
+}
+```
+
+## useMultiSelect Hook implementation in a `dialog` MultiSelect
```

Comment:
The difference between `listbox` and `dialog` is not obvious until you read the code. That's fine but let's make it clearer to devs by writing a short description. Add that for the `dialog` `MultiSelect` component, the implementation now uses the `onMixedStateChange` function.

Reply:
I added a small description, let me know what you think.